client/app/bundles/momentCard/components/MomentCardViewers.jsx

```diff
@@ -0,0 +1,20 @@
+// @flow
+import React from 'react';
+
+type MomentCardViewersState = {};
+
+type MomentCardViewersProp = {};
+
+export default class MomentCardViewers extends
+  React.Component<MomentCardViewersProp, MomentCardViewersState> {
+  props: MomentCardViewersProp;
+  state: MomentCardViewersState;
+
+  render() {
+    return (
+      <div>
+        Viewers
```

Comment:
You should use the i18n key here

Reply:
You can omit MomentCardViewersState as the second parameter (it's optional).

On the following lines, you can omit `props: MomentCardViewersProp` and `state: MomentCardViewersState` because they should already be defined as part of React.Component< MomentCardViewersProp>.